packages/manager/modules/pci/src/components/project/flavors-list/flavor-group.class.js

```diff
@@ -9,9 +9,12 @@ import { DEFAULT_OS } from './flavors-list.constants';
 
 export default class FlavorGroup {
   constructor(flavors) {
-    Object.assign(this, omit(find(flavors, { osType: DEFAULT_OS }), [
-      'regions', 'id', 'osType', 'planCodes',
-    ]));
+    Object.assign(
```

Comment:
```suggestion
    assign(
```

(my understanding is that we should be using `Lodash` most of the time)

Reply:
@FredericEspiau `Lodash` is useful for defensive pattern (e.g: if I'm not sure the value I'll be giving will be defined) but here `this` is defined so no need. 
To be honest I don't know what's best but with using `Object.assign` it avoids importing more things to the class